concent_api/common/tests/test_decorators.py

```diff
@@ -21,10 +21,11 @@
 from core.decorators import handle_errors_and_responses
 from core.decorators import require_golem_auth_message
 from core.exceptions import Http400
+from core.exceptions import SCINotSynchronized
 from core.tests.utils import ConcentIntegrationTestCase
 from core.tests.utils import parse_iso_date_to_timestamp
 
-(CONCENT_PRIVATE_KEY, CONCENT_PUBLIC_KEY)   = generate_ecc_key_pair()
+(CONCENT_PRIVATE_KEY, CONCENT_PUBLIC_KEY) = generate_ecc_key_pair()
```

Comment:
Why not using keys that are already available in `ConcentIntegrationTestCase` class?